src/templates/rule.js

```diff
@@ -111,6 +108,11 @@ export default ({ data }) => {
 								Propose a change to the rule
 							</a>
 						</li>
+						<li>
```

Comment:
I think we should put the Changelog link close to the Issues link since I feel they are closely related.